server_development/topics/providers.adoc

```diff
@@ -391,6 +438,8 @@ Once you have a JAR file with a descriptor and the scripts you want to deploy, y
 
 ==== Using {project_name} Administration Console to upload scripts
 
+NOTE: Ability to upload scripts through the admin console is deprecated and will be removed in a future version of {project_name}
+
 By default, the {project_name} Administration Console won't allow administrators to upload scripts to the server. The reason is that
 script execution can potentially harm the system due to the execution of malicious scripts. In order to prevent different types of
 attacks and lower the risks when running scripts at runtime, administrators should prefer deploying scripts directly to the server using a JAR file.
```

Comment:
Administrators cannot upload scripts to the server. This default behavior prevents potential harm to the system in case malicious scripts are accidentally executed. Administrators should always deploy scripts directly to the server using a JAR file to lower prevent attacks when you run scripts at runtime.

Reply:
Fixing